src/components/NavigationBar/index.js

```diff
@@ -321,6 +325,7 @@ class NavigationBar extends Component {
         key={name}
         checked={values && values.indexOf(name) > -1}
         value={name}
+        onClick={clicked}
```

Comment:
onClick expects a function. Not an object. Please check your implementation again.

Reply:
Working on it again sir.